development/docker-compose.celery.yml

```diff
@@ -0,0 +1,17 @@
+---
+version: "3.4"
+services:
+  celery:
+    image: "nautobot-chatops-plugin/nautobot:${NAUTOBOT_VER}-py${PYTHON_VER}"
+    env_file:
+      - "dev.env"
+      - "creds.env"
+    tty: true
+    entrypoint: "nautobot-server celery worker -B -l INFO"
```

Comment:
Nitpick: `-B` isn't actually needed (it causes the worker to self-host Celery Beat, but we're not actually using that feature in 1.1, and it'll be different in 1.2, see https://github.com/nautobot/nautobot/pull/805/files#diff-338f8adcdd0eb410e4af13c5ab840ae9026cdecefae4a0a421579675749b9c2eR37). We should change this in Nautobot core (where you presumably copied this from) as well - I'll get an issue opened.

Reply:
https://github.com/nautobot/nautobot/issues/825 FYI.

Reply:
Thanks! I actually copied it from the onboarding plugin